cli/src/pcluster/cli/commands/configure/easyconfig.py

```diff
@@ -341,7 +341,18 @@ def _create_vpc_parameters(scheduler, head_node_instance_type, compute_instance_
     vpc_list = vpc_and_subnets["vpc_list"]
     if not vpc_list:
         print("There are no VPC for the given region. Starting automatic creation of VPC and subnets...")
-    if not vpc_list or prompt("Automate VPC creation? (y/n)", lambda x: x in ("y", "n"), default_value="n") == "y":
+    if (
+        not vpc_list
+        or prompt(
+            "The creation of the VPC will result in minimal charges for NAT gateway \n"
```

Comment:
Would it be better to say `standard` than `minimal` since minimal is subjective?

Reply:
That could be, but I don't want to alarm people with warning.  A NAT gateway cost ~$1 a day